### ISSUE TITLE
Add MedicationRequest Category 

### ISSUE THREAD

**Issue by ssmillard** [cds-hooks/sandbox#136]
Would it be possible to add ability to populate the MedicationRequest category for draft submissions on RX View
https://www.hl7.org/fhir/valueset-medicationrequest-category.html

**Comment by dennispatterson** [comment]
i.e. when the FHIR server is stu3 or r4, you'd like a drop-down with the category options? (it varies slightly between the two FHIR versions)

Just for insight into your use-case, is the first step just to have *any* category sent just to ensure the field is populated?

**Comment by ssmillard** [comment]
Correct, A drop down with inpatient, outpatient, and community.  I believe those are consistent between versions 3 +4.  Our use cases can have different outcomes based on the category.
